src/ekka_mnesia.erl

```diff
@@ -399,3 +397,17 @@ find_upstream_node([Shard|Rest]) ->
         disconnected ->
             find_upstream_node(Rest)
     end.
+
+-spec call_backend(atom(), list()) -> term().
+call_backend(Function, Args) ->
+    case {backend(), ekka_rlog:role()} of
+        {mnesia, core} ->
+            apply(mnesia, Function, Args);
+        {mnesia, replicant} ->
+            exit(plain_mnesia_transaction_on_replicant);
```

Comment:
Use `error` rather than `exit`

Reply:
This is copy/paste from the mnesia code, I don't know why they used exit here. Will fix